app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_for post do |form| %>
+  <%= form.label :message%>
+  <%= form.text_field :message, placeholder: "WTF You Thinking?!" %>
+  
+  <%= form.label :image %>
+  <%= form.file_field :image %>
+    
+  <% post.errors.full_messages_for(:message).each do |error_message| %>
+    <div><%= error_message %></div>
```

Comment:
What are the error messages for? Unsure what establishes as an error when adding a post, and where it is defined?